package.json

```diff
@@ -51,6 +52,22 @@
   },
   "devDependencies": {
     "cypress": "^3.0.1",
+    "eslint": "^4.19.1",
+    "eslint-config-airbnb": "^17.0.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-prettier": "^2.6.0",
+    "eslint-plugin-react": "^7.10.0",
+    "husky": "^0.14.3",
```

Comment:
Remove `husky`. Another PR already does this. Its not needed

Reply:
I have removed the husky dependency